docs/decisions/0008-abandon-reactstrap.rst

```diff
@@ -0,0 +1,36 @@
+5. Abandon Reactstrap
+---------------------
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+As proposed in ADR-0005 we experimented with Reactstrap components in the Profile micro-frontend application. On the plus side, Reactstrap is clean to look at, easy to use and removes the need to know Bootstrap class names to get started.
+
+This comes at the cost of abstracting away raw html and introducing yet another API to learn and remember. In some cases, Reactstrap components serve little purpose: The <Form /> component, for example, offers just one boolean prop (<Form inline />) to toggle the bootstrap class name "form-inline".
```

Comment:
nit: be consistent with the naming of Bootstrap, capitalize or not. Earlier, it's capitalized but here it's not.